### PR TITLE
remove tablet view and fix responsiveness issues

### LINKED ISSUE #1950: Wins page - make the tablet like the mobile view
### Overview
@daniellex0 indicated in the issue below that we should have the wins page look like the mobile view in tablet (below 960px)
- #1933

### Action Items
- [x] discover where in the css and html the screen size is being used to determine the layout of the wins page
- [x] Make it so the current mobile view layout is expanded to include tablet view
- [x] Check to see that the see more see less functionality still works

### Resources/Instructions
IMPORTANT - This issue is working off of a feature branch!
Please follow this wiki before creating a new branch!
[how to work off a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)

### PR DESCRIPTION
**Pull request by jbubar**
Fixes #1950

### What changes did you make and why did you make them ?

  - Removed the tablet view by expanding the mobile view to desktop

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screen Shot 2021-07-15 at 11 30 02 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screen Shot 2021-07-15 at 11 27 32 PM

</details>

### CODE REVIEW

**Comment by jbubar** [comment]
If you do not already have a `wins-feature-1` branch. please use this guide to create one...
https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch

**Review by erikaBell — CHANGES_REQUESTED** [review]
Great new tablet view! (or lack there of)

I saw on Issue #1950 one of the check items was to make sure see less/see more functionality still worked, however I have found a bug where the "see more" functionality stops working, and "see less" gets stuck on desktop view, if it was previously opened on smaller screen size. Everything returns to functionality when the page is refreshed. 

Video below showcases bug.

<details>
<summary>see more / see less bug</summary>


https://user-images.githubusercontent.com/67438372/126047127-e2b5131f-3447-45ac-b183-8f1a6bcddd33.mov


</details>

<details>
<summary>separate, CSS formatting</summary>

Screen Shot 2021-07-17 at 11 38 17 AM

</details>

**Comment by jbubar** [comment]
Still working on this. I will ask for another review after I am done. thanks A bunch @erikaBell

File: pages/wins/wins.html [hunk 1]
```diff
@@ -472,18 +480,7 @@
...
-			  document.querySelector(".home-getting-started-container.content-section.section-hack-nights.wins-hero").inert = false;
-			  document.querySelector(".sr-only-focusable").inert = false;
-		  }
-		  else {
-			  document.querySelector(".main-header").inert = true;
-			  document.querySelector(".content-section.projects").inert = true;
-			  document.querySelector(".main-footer").inert = true;
-			  document.querySelector(".home-getting-started-container.content-section.section-hack-nights.wins-hero").inert = true;
-			  document.querySelector(".sr-only-focusable").inert = true;
+			  hideOverlay(e)
```
Comment (jbubar) [inline pages/wins/wins.html:479]: This fixed the unclickability on desktop

File: _sass/components/_wins-page.scss [hunk 2]
```diff
@@ -21,6 +21,24 @@
     max-height: 75px;
 }
 
+.win-list{
```
Comment (jbubar) [inline _sass/components/_wins-page.scss:24]: This was to remove tablet view

File: pages/wins/wins-card-template.html [hunk 3]
```diff
@@ -26,6 +26,6 @@
         </div>
       </div>
     </div>
-    <div class="wins-icon-container"></div>
+    <div class="wins-icon-container" id="icons-0"></div>
```
Comment (jbubar) [inline pages/wins/wins-card-template.html:26]: Added this to make the div more accessible... so you dont have to do element.parent.parent.parent.nextsibling or whatever

File: pages/wins/wins.html [hunk 4]
```diff
@@ -280,11 +280,12 @@
   }
 
  
-	function insertIcons(cloneCardTemplate, cardClass, cardString) {
+	function insertIcons(cardSelector, cardString, viewType, cloneCardTemplate = document) {
```
Comment (jbubar) [inline pages/wins/wins.html:280]: Refactored this function to work insert icons on both wins cards, and the overlay

File: pages/wins/wins.html [hunk 5]
```diff
@@ -374,46 +374,55 @@
...
 		}
 		addSeeMore()
 
 		new ResizeObserver(addSeeMore).observe(winText)
 	})
-  }
+}
 
-  function seeMore(id){
-  	screenWidth = window.innerWidth;
```
Comment (jbubar) [inline pages/wins/wins.html:407]: changed this, because we should not be using global variables

File: pages/wins/wins.html [hunk 6]
```diff
@@ -374,46 +374,55 @@
...
   		updateOverlay(id);
   	}else{
 		toggleSeeMoreLess(id);
   	}
-  }
+}
 
   //Adds event listener to badge icons when viewers interface is below desktop size
   //Click to toggle "see more" and "see less" to expand and collapse wins text
+  document.addEventListener("DOMContentLoaded", winsBelowDesktop);
```
Comment (jbubar) [inline pages/wins/wins.html:419]: Added back in the clickability of the icons in mobile view

File: pages/wins/wins.html [hunk 7]
```diff
@@ -516,14 +513,16 @@
...
+		// insertOverlay(overlayInfo, data[i][win])
+		const iconContainer = insertIcons('#overlay-info', data[i][win], 'overlay')
 
   		const overlayProjectCard = document.querySelector('#overlay-project-card');
   		overlayProjectCard.parentNode.classList.add("display-initial");
   	}
 
 
 	function hideOverlay(e) {
+		console.log("in the hide overlay")
```
Comment (adam-abundis) [inline pages/wins/wins.html]: You may want to comment out this console.log

**Review by adam-abundis — APPROVED** [review]
@jbubar. Great job. I like the DRY approach you took to refactor the `insertIcons` function so it to work for overlay/mobile view.

 I have verified the tablet view now uses the mobile view. The show more/show less functionality also works in desktop, tablet, and mobile view properly.

My only recommendation would be to comment out line 525 within `pages/wins/wins.html` (console.log within the `hideOverlay` function. Other than that, this is ready to be merged.

**Comment by averdin2** [comment]
@jbubar I am not sure if this issue was already there before you made changes, but I am seeing that when in tablet/mobile view if you click on a badge icon it opens up the see more view. If you click on the badge icon again, it closes the see more view and shows the text bubble/tooltip. This behavior seems a bit weird, I am not sure if we want the page to act this way. See the gif below for the visual.

<details>
<summary>Mobile View Tooltip Behavior</summary>

Screen Recording 2021-07-19 at 4 27 36 PM
</details>

**Review by marcobarrera — APPROVED** [review]
The responsiveness is working. In tablet view (under 960px), it now has the same view as mobile. The see more/less links are also working fine on all cards.

**Comment by erikaBell** [comment]
@jbubar Great, see more/less is working as expected now. I do agree with @averdin2's comment above. But that could be a separate issue. Or should we check in with @daniellex0 first?

Either way, nice work 🐸

If we want to keep moving, I feel good to go ahead and merge. I think you were still working on the CSS format though, @jbubar?

**Comment by jbubar** [comment]
@averdin2 cool idea. I will do that right now. @erikaBell Thanks for reviewing this! yes, I think I would like to put the styling in another issue. it will be a bit of a refactor.

File: pages/wins/wins.html (hunk 7 above)
Comment (jbubar) [reply pages/wins/wins.html]: Thank you!!!

**Review by erikaBell — APPROVED** [review]
lg

**Pull request merged**